### DISCUSSION TITLE
Writing to the $GITHUB_STEP_SUMMARY with the core npm package

### DISCUSSION THREAD

**Discussion by giscus** [rajbos/rajbos.github.io#36] [Comments]
# Writing to the $GITHUB_STEP_SUMMARY with the core npm package

Writing to the $GITHUB_STEP_SUMMARY with the core npm package

https://devopsjournal.io/blog/2023/06/08/GITHUB_STEP_SUMMARY

**Comment by fijiaaron** [comment]
Thanks for breaking down how to customize the GitHub Actions summary table.

**Reply by rajbos** [reply]
Thanks for letting me know @fijiaaron ! I ran into this myself a couple of times as well 😄
